packages/api/lib/ems.js

```diff
@@ -101,6 +102,22 @@ async function getExpiredS3Objects(bucket, prefix, retentionInDays) {
     .map((s3Object) => ({ Bucket: bucket, Key: s3Object.Key }));
 }
 
+async function retrievePrivateKey() {
+  const privateKeyFile = process.env.ems_privateKey || 'ems-private.pem';
+
+  let privateKey;
+  try {
+    privateKey = await getTextObject(process.env.system_bucket, `${process.env.stackName}/crypto/${privateKeyFile}`);
```

Comment:
This line could just be `return await getTextObject(...)`, which would simplify it a bit.

Reply:
Not worth holding up the approval over, it's a small thing. Or, as Menno would say, something about 🐜 .

Reply:
I tried `return getTextObject` and it wouldn't catch the error. Did it need an `await` in there?

Reply:
Yeah, the `await` is what makes the difference.